docs/reference/search/rrf.asciidoc

```diff
@@ -1,6 +1,423 @@
 [[rrf]]
 === Reciprocal rank fusion
 
-Reciprocal Rank Fusion (RRF) is a simple method to combine document result sets
-from multiple queries where the queries' document scores may be unrelated.
+NOTE: Reciprocal rank fusion requires a Platinum subscription or higher.
 
+IMPORTANT: Reciprocal rank fusion is in technical preview. Features in technical
+preview may be changed or removed in a future release. Elastic will apply best
+effort to fix any issues, but features in technical preview are not subject to
+the support SLA of official GA features.
```

Comment:
```suggestion
preview::[]
```

Reply:
You can use [the `preview::[]` admonition](https://github.com/elastic/docs#using-the-technical-preview-admonition) to ensure the message is consistent across the entire documentation.

Reply:
Changed.